libstemmer/modules.txt

```diff
@@ -34,6 +34,7 @@ basque          UTF_8,ISO_8859_1        basque,eu,eus,baq
 swedish         UTF_8,ISO_8859_1        swedish,sv,swe
 tamil           UTF_8                   tamil,ta,tam
 turkish         UTF_8                   turkish,tr,tur
+estonian		UTF_8,ISO_8859_1		estonian,et,est
```

Comment:
Please keep the sorting order, i.e. insert this between English and Finnish. See 0e0d307b082c0c5758ea7f89bb9b3d3dd9b215e1.

Reply:
Fixed by eb1d16a2acc7b7e6e9c6770a53ec2811da65f127.